### PR TITLE
PullRequest_ginkgo

### PR DESCRIPTION
**Pull request by santosh653**
creating ginkgo.travis.yml with proposed changes.,

### CODE REVIEW

File: travis-ymls/ginko.travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : ginkgo
+# Source Repo         : https://github.com/onsi/ginkgo
+# Travis Job Link     : https://travis-ci.com/github/santosh653/ginkgo
```
Comment (seth-priya) [inline travis-ymls/ginko.travis.yml:5]: I see that all the ppc64le job are failing?

File: travis-ymls/ginko.travis.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - amd64
+  - ppc64le
+language: go
+go:
+  - 1.14.x
+  - 1.15.x
+  - tip
+jobs:
+ exclude:
+  - go: tip
```
Comment (seth-priya) [inline travis-ymls/ginko.travis.yml:23]: Excluding older versions is fine but I don't think we should be excluding "tip"

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: travis-ymls/ginko.travis.yml (hunk 2 above)
Comment (santosh653) [reply travis-ymls/ginko.travis.yml:23]: @seth-priya Now the code is running as suggested by you., Can you please review?

Comment (seth-priya) [reply travis-ymls/ginko.travis.yml:23]: as far as I can see we are still excluding "tip" for ppc64le, am I missing something?

**Comment by santosh653** [comment]
@seth-priya the code is running fine only 1.14.x is excluded for ppc64le. Please find below the Travis/github link. 
https://travis-ci.com/github/santosh653/ginkgo
https://github.com/santosh653/ginkgo/blob/master/.travis.yml

Thank you !!

**Comment by seth-priya** [comment]
@santosh653 as I have said multiple times before, I still do not see the changes that you are talking about reflected in this file or commit, it still shows that "tip" is excluded - please review/cross-check that you are updating the correct branch etc. @gururajrkatti , could you please have a second look.

**Comment by santosh653** [comment]
@seth-priya Apologies for the confusion, I am closing this Pull Request & opening a new one with all that is required.
Thank you !!

**Pull request closed without merging**